docs/software-platforms/aws/index.md

```diff
@@ -3,24 +3,21 @@
 We use Amazon Web Services (AWS) to grant contributors access to data and Linux virtual computers.
 
 !!! note
-    See [Getting Access to Resources](../../getting-started/accessing-resources/index.md) for more information about AWS access.
+    See [Getting Access to Resources](../../getting-started/accessing-resources/index.md) for more information about getting AWS access.
 
-## Joining IAM Identity Center
+    Once we have created an AWS account for you, you can proceed to [set up your account](./joining-aws.md).
+    Note that your use of AWS services described in this section counts towards your [monthly budget](../../getting-started/accessing-resources/getting-access-to-compute.md#monthly-budget).
 
-When we create an account for you, we will also create a user for you with the username `researcher-{your github handle}`.
 
-You will receive an email from `no-reply@login.awsapps.com` with the subject "Invitation to join AWS IAM Identity Center (successor to AWS Single Sign-On)."
+## S3: AWS for data
 
-You must accept this invitation to gain access to your AWS account and to download project data from S3.
+We use [AWS S3](https://aws.amazon.com/s3/) to store project data in folders referred to in S3 as "buckets."
+Please refer to [our documentation on accessing project data](../../getting-started/accessing-resources/getting-access-to-data.md#accessing-data-on-s3) for more information.
 
-To accept the invitation, please take the following steps.
+We also use S3 as central location for you to share your result files with us before [filing a pull request](../../contributing-to-analyses/creating-pull-requests/index.md).
+All contributors will have a designated [researcher bucket](working-with-s3-buckets.md) where results can be synced and shared as part of [code review](../../contributing-to-analyses/pr-review-and-merge/index.md).
 
-1. Note the following information in the invitation email: **Your AWS access portal URL**.
-   You will need this value to [configure AWS CLI locally](../../technical-setup/environment-setup/configure-aws-cli.md) and to [log into the console to use Lightsail for Research](STUB_LINK).
-   It is also helpful to bookmark this URL so you can easily return to it.
 
-2. Click the `Accept Invitation` button in the email.
+## LSfR: AWS for virtual computing
 
```

Comment:
I think I would add a sentence here about why you would want to use lightsail (access to computing resources so you don't have to use your own). And then a bulleted list of what you will need to set up with links to the docs. 

I would probably do the same with a bulleted list for S3 with links to each of the docs?

Reply:
> a bulleted list for S3 with links to each of the docs

For this, I'll link both the one doc in this section, and relevant docs pages from other sections.